src/dstore/plugins/cortx/cortx_dstore.c

```diff
@@ -576,5 +576,5 @@ const struct dstore_ops cortx_dstore_ops = {
 	.io_op_submit = cortx_ds_io_op_submit,
 	.io_op_wait = cortx_ds_io_op_wait,
 	.io_op_fini = cortx_ds_io_op_fini,
-	.obj_get_bsize = cortx_ds_obj_get_bsize,
+	/* .obj_get_bsize = cortx_ds_obj_get_bsize, // deprecated */
```

Comment:
Should we just remove all the references of this function rather than commenting it out?

Reply:
Addressed.